client/src/app/containers/NewGameForm/index.jsx

```diff
@@ -1,7 +1,116 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import { useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { useInjectReducer, useInjectSaga } from 'redux-injectors';
+
+import { createGameSaga } from './saga';
+import { sliceKey, reducer, actions } from './slice';
+
+const useStyles = makeStyles(theme => ({
+  title: {
+    fontSize: '2rem',
+  },
+  paper: {
+    marginTop: theme.spacing(8),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(1),
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
 
 const NewGameForm = () => {
-  return <div>Make a new game here</div>;
+  // hooks
+  useInjectReducer({ key: sliceKey, reducer: reducer });
+  useInjectSaga({ key: sliceKey, saga: createGameSaga });
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const [opp1ID, setOpp1ID] = useState('');
+  const [opp2ID, setOpp2ID] = useState('');
+  const [teammateID, setTeammateID] = useState('');
+
+  // event handlers
+  const onSubmitLoginForm = event => {
+    event.preventDefault();
+    dispatch(actions.createGame(opp1ID, opp2ID, teammateID, history));
+  };
+  const onOpp1IDChange = event => {
+    setOpp1ID(event.target.value);
+  };
+  const onOpp2IDChange = event => {
+    setOpp2ID(event.target.value);
+  };
+  const onTeammateIDChange = event => {
+    setTeammateID(event.target.value);
+  };
```

Comment:
If you _wanted_ to, there's a pretty slick way to do this:
```jsx
const [formData, setFormData] = useState({
  id1: '',
  id2: '',
  teammateID: '',
});
const onFormDataChange = e => {
  setFormData({ ...formData, [e.target.name]: e.target.value });
};
```

The `name` prop on each `TextField` just has to correspond to what you call them in the initial `formData`.